zebra-network/src/protocol/external/inv.rs

```diff
@@ -45,8 +46,21 @@ pub enum InventoryHash {
     /// [auth_digest]: https://zips.z.cash/zip-0244#authorizing-data-commitment
     /// [zip239]: https://zips.z.cash/zip-0239
     /// [bip339]: https://github.com/bitcoin/bips/blob/master/bip-0339.mediawiki
-    // TODO: Actually handle this variant once the mempool is implemented
-    Wtx([u8; 64]),
+    // TODO: Actually handle this variant once the mempool is implemented (#2449)
```

Comment:
We haven't finished this TODO, but we won't need to change this code again as part of #2449.

```suggestion
```